common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -145,7 +135,7 @@ trait ABTestSwitches {
     "Bootstrap the AB test framework to use the Epic to thank readers who have already supported the Guardian",
     owners = Seq(Owner.withGithub("Mullefa")),
     safeState = On,
-    sellByDate = new LocalDate(2017, 8, 1),
```

Comment:
Could this go to a far-future date? Can't see us switching this off anytime soon